samples/go/hmac/signature/signatureGenerator.go

```diff
@@ -0,0 +1,39 @@
+package signature
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/base64"
+	"net/url"
+)
+
+const (
+	Algorithm       = "algorithm=\"hmac-sha1\","
+	DatePrefix      = "date: "
+	Headers         = "headers=\"date x-mod-nonce\","
+	SignaturePrefix = "signature=\""
+	SignatureSuffix = "\""
+	Nonce           = "\nx-mod-nonce: "
```

Comment:
I'd probably prefer to see this split into `Newline` and `Nonce`

Reply:
good shout, will amend